CoreFoundation/Stream.subproj/CFSocketStream.c

```diff
@@ -110,7 +110,7 @@ static struct {
 
```

Comment:
taking a guess here on adding the Linux targets, but assuming ```CFNETWORK_LIBRARY_PATH``` is found from ```__CFgetenvIfNotRestricted```  it should work?
if not we may need to do a 
```
const char *path = __CFgetenvIfNotRestricted("CFNETWORK_LIBRARY_PATH");
	if (!path) {
#if DEPLOYMENT_TARGET_MACOSX  
        path = "/System/Library/Frameworks/CFNetwork.framework/CFNetwork";
#elif DEPLOYMENT_TARGET_LINUX 
        path = ".../Linux/CFNetwork";
#endif
	}
```
@LoganWright can you test this to see if this works, i still don't have Linux build working locally